### PR TITLE
Bug 1957198: Remove misspelled console-operator condition

### PR DESCRIPTION
**Pull request by jhadvig**
/assign @spadgett

### CODE REVIEW

File: pkg/console/starter/starter.go [hunk 1]
```diff
@@ -324,19 +325,20 @@ func RunOperator(ctx context.Context, controllerContext *controllercmd.Controlle
...
-	//)
+	staleConditionsController := staleconditions.NewRemoveStaleConditionsController(
+		[]string{
+			// If a condition is removed, we need to add it here for at least
+			// one release to ensure the operator does not permanently wedge.
+			// Please do something like the following:
+			//
+			// example: in 4.x.x we removed FooDegraded condition and can remove
+			// this in 4.x+1:
+			// "FooDegraded",
```
Comment (spadgett) [inline pkg/console/starter/starter.go:336]: Can you update this comment to explain that the removed condition was misspelled and what release that can be removed in?

**Comment by jhadvig** [comment]
@spadgett comment addressed. PTAL

**Review by spadgett — APPROVED** [review]
/lgtm

**Review by spadgett — APPROVED** [review]
/lgtm

**Pull request merged**